core/appcontainer/AppContainerModel.js

```diff
@@ -27,25 +27,26 @@ export class AppContainerModel {
     //------------
     // Sub-models
     //------------
-    aboutDialogModel = new AboutDialogModel();
-    exceptionDialogModel = new ExceptionDialogModel();
-    optionsDialogModel = new OptionsDialogModel();
-    feedbackDialogModel = new FeedbackDialogModel();
-    impersonationBarModel = new ImpersonationBarModel();
-    loginPanelModel = new LoginPanelModel();
-    messageSourceModel = new MessageSourceModel();
-    toastSourceModel = new ToastSourceModel();
-    themeModel = new ThemeModel();
-    refreshContextModel = new RefreshContextModel();
+    @managed aboutDialogModel = new AboutDialogModel();
```

Comment:
Should these decorators be moved to the preceding line to be consistent with the rest of the code? Is that the standard we are trying to follow in general? I wonder if it would be worth adding to the eslint config if possible?

Admittedly this set of declarations is nice and symmetrical with the decorator on the same line ...